content/_books/index.html

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
If you moved this file to `/books.md` then it would get the same url (`/books/`) but will avoid the problem of it being in the collection itself (you could remove the `{% if`)